resources/keepalived-docker/manage-keepalived.sh

```diff
@@ -5,12 +5,4 @@ export interface=$PROVISIONING_INTERFACE
 sed  -i "s~INTERFACE~$interface~g" /etc/keepalived/keepalived.conf
 sed  -i "s~CHANGEIP~$assignedIP~g" /etc/keepalived/keepalived.conf
 
-while true;do
-        /etc/init.d/keepalived start
-	if (ip a show "$interface" |grep "inet $assignedIP"); then
-           echo "waiting for VIP to be assigned on interface $interface"
-        else
-           exit 0
-        fi
-        sleep 1
-done
+exec /usr/sbin/keepalived -n -l
```

Comment:
how do you verify whether the IP was assigned to endpoint or not?

Reply:
you don't. If keepalived does not start properly, it will exit, then K8S will restart it

Reply:
how should we test it whether its working properly or not? have you tested it locally?

Reply:
I'm modifying the CI tests right now

Reply:
metal3-io/metal3-dev-env#425 and metal3-io/project-infra#137

Reply:
> you don't. If keepalived does not start properly, it will exit, then K8S will restart it

That sounds fine, but just to be clear... we're not expecting this to e.g. always fail when another host has the VIP are we?  I don't think so, but I don't actually understand what the previous code was trying to do. I do know that CrashLoopBackoff is not the mechanism you want to use for failover ;)

Reply:
no, Keepalived will keep running,  in case the master instance fails. It should not exit. I don't recall why the loop was there in the first place, so I just removed it.

Reply:
I was referring to an actual failure to start, like a misconfiguration or a temporary unavailable interface, where we actually want a failure to start to appear and not be hidden by a loop.

Reply:
The loop was there because it failed to restart itself I believe back then , it was a bug from keepalived if I recall correctly so we went on with the loop